runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/util/common/worker/WorkProgressUpdaterTest.java

```diff
@@ -165,8 +165,7 @@ public void updatesBeforePeriodicCheckpoint() throws Exception {
     // Set the initial lease expiration to 20s so that the first update occurs at 10s, ie before
     // the periodic checkpoint.
     initialLeaseExpirationMs = clock.currentTimeMillis() + 20 * 1000L;
-    when(progressHelper.reportProgress(any(NativeReader.DynamicSplitResult.class)))
-        .thenReturn(4 * 1000L); // Next update at 14s.
+    when(progressHelper.reportProgress(null)).thenReturn(4 * 1000L); // Next update at 14s.
```

Comment:
This change looks wrong (and all the bits below)

Reply:
Same response as above :cry: If we want to change this, we have to fix the mocks to not call this method with `null`, but I am unfamiliar with the code to easily figure out how that works.

Reply:
Turns it it *is* correct. The test in question makes sure that the `reportProgress` is invoked with `null`. Especially since line 177 is `verify(progressHelper, times(1)).reportProgress(null);` So I think this change is actually correct.

Reply:
Nice. In that case, you've made it more clear :-)

Reply:
Got it